modular_zubbers/code/modules/bow_nerf/arrow.dm

```diff
@@ -0,0 +1,110 @@
+/obj/projectile/bullet/arrow
+	damage = 20
+	armour_penetration = 0
+	weak_against_armour = TRUE
+	wound_bonus = CANT_WOUND
```

Comment:
I'd set this so that damage+wound_bonus = 0, so that they still wound occasionally. Or rather than 0 even 5-15 if you feel like it, mosin does 15- we'd have to test how strong this is.